Makefile

```diff
@@ -0,0 +1,25 @@
+.PHONY: test hh_autoload composer
+
+COMPOSER=~/.cache/composer/composer.phar
+
+test: hh_autoload
+	./vendor/bin/hacktest tests
+
+lint: hh_autoload
+	./vendor/bin/hhast-lint
+
+autoload:
+	./vendor/bin/hh-autoload
+
+composer-fetch:
+	test -f $(COMPOSER) || mkdir -p `dirname $(COMPOSER)`
```

Comment:
This needs to be whitespace-safe, both for space-in-homedir and space-in-checkout dir, and isn't